projects/wave-core-new/src/lib/layers/legend/legend-raster/raster-legend.component.html

```diff
@@ -12,6 +12,15 @@
         </tr>
     </ng-template>
 
+    <ng-template [ngIf]="layer.symbology.colorizer.isDiscrete()">
+        <tr *ngFor="let breakpoint of layer.symbology.colorizer.getBreakpoints(); let isFirst = first">
```

Comment:
if you don't use the `isFirst` variable you can remove it

Reply:
I removed it e17e0f5a4797d018cde265ee8cd07d444dbe25e6